src/components/reviews/ReviewsList.jsx

```diff
@@ -1,12 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import ReviewTile from './ReviewTile.jsx';
-import styles from "./reviews.module.css";
-import { useProductContext } from "../../context/ProductContext.jsx";
+import ReviewForm from './ReviewForm.jsx';
+import styles from './reviews.module.css';
+import modalStyle from '../questions/questions.module.css';
+import { useProductContext } from '../../context/ProductContext.jsx';
 
 const ReviewsList = () => {
   const {reviews} = useProductContext();
+  const [modalIsActive, setModalState] = useState(false);
   const reviewsList = reviews.results;
 
+  const startReview = () => {
+    setModalState(true);
+  }
+
+  const cancelReview = () => {
+    setModalState(false);
+  }
+
```

Comment:
to DRY the code, maybe you can combine the startReview() and cancelReview() together :

const handleModalActive = () => {
        setModalState(!modalIsActive)
}